### ISSUE TITLE
Getting error while exporting SOAP API's

### ISSUE THREAD

**Issue by kunwardhwaj** [Axway-API-Management-Plus/apim-cli#48]
**API-Mgr. CLI version**
> Please make sure to use the latest available version before reporting a bug.

1.1.0

**API-Manager and Service-Pack Version**
7.6.2/SP3

**Expected behavior**

should be able to export SOAP API's as well.

**Actual behavior**

getting error while exporting SOAP API's. Works fine for REST APIs.

**Log-Output in Debug**
```
apim api get -s dev -id 676834c0-f3f8-4502-97ac-c19509b58edd --o json -t C:\axway\apis

----------------------------------------------------------------------------------------
API-Manager CLI: 1.1.0

To report issues or get help, please visit:
https://github.com/Axway-API-Management-Plus/apim-cli
----------------------------------------------------------------------------------------
Module: API - E X P O R T / U T I L S (1.1.0)
----------------------------------------------------------------------------------------
928    INFO     APIManagerAdapter| Successfully connected to API-Manager (7.6.2 SP3) on: https://xxxxxxxxxxxxxx:8075
6130   ERROR         APIExportApp| Can't read Swagger-File.
com.axway.apim.lib.errorHandling.AppException: Can't read Swagger-File.
        at com.axway.apim.adapter.apis.APIManagerAPIAdapter.addOriginalAPIDefinitionFromAPIM(APIManagerAPIAdapter.java:491)
        at com.axway.apim.adapter.apis.APIManagerAPIAdapter.getAPIs(APIManagerAPIAdapter.java:125)
        at com.axway.apim.APIExportApp.runExport(APIExportApp.java:104)
        at com.axway.apim.APIExportApp.export(APIExportApp.java:48)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.axway.apim.cli.APIManagerCLI.run(APIManagerCLI.java:138)
        at com.axway.apim.cli.APIManagerCLI.main(APIManagerCLI.java:76)
Caused by: com.axway.apim.lib.errorHandling.AppException: Can't handle API specification. No suiteable specification implementation found.
        at com.axway.apim.api.definition.APISpecificationFactory.getAPISpecification(APISpecificationFactory.java:52)
        at com.axway.apim.adapter.apis.APIManagerAPIAdapter.addOriginalAPIDefinitionFromAPIM(APIManagerAPIAdapter.java:488)
        ... 9 more
```

**Comment by cwiechmann** [comment]
@kunwardhwaj - BTW: In the meantime you ca download version [1.2.0 RC1](https://github.com/Axway-API-Management-Plus/apim-cli/releases/tag/apimcli-1.2.0-RC1) which implements some kind of a fallback for unkown API-Specs. This means, you should be able to export your SOAP-Service, but the spec-file is then names .txt instead of .wsdl

**Comment by cwiechmann** [comment]
Hi @kunwardhwaj, Today I had the time to try to replicate this issue on my side, but had no luck. I have imported the SOAP-Service and successfully exported it afterwards. I used an API-Management 7.6.2 and 7.7-May-Release. Both are working okay. 
Did you had the chance to test with version 1.2.0 RC1 ? Which supports that "fallback" to a simple text file. With that, you get at least the APIs/Services exported even if the SOAP-File in then called .txt.
